adapter/tools/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# For building this image, you need to be in the project
```

Comment:
not sure if we need to put this dockerfile here

Reply:
This was an easy way to get an image with dbt-fal in it.